docs/field_projectid.rst

```diff
@@ -75,19 +75,46 @@ Title of the project, award or grant (occurrence: 0-1).
 Example
 ~~~~~~~
 
-An example utilizing all fields:
+Examples utilizing all fields:
 
 .. code-block:: xml
    :linenos:
 
    <oaire:fundingReferences>
     <oaire:fundingReference>
-     <oaire:funderName>European Commission</oaire:funderName>
+     <oaire:funderName>European Commission</datacite:funderName>
```

Comment:
this has a malformed xml element. The with a wrong end tag - </datacite:funderName>

Reply:
Correct is correct also in the branch "v4.1-draft".